### PR TITLE
Upgrade PostCSS Preset Env

### LINKED ISSUE #326: Update postcss-preset-env to v9
### Is your enhancement related to a problem? Please describe.

Update postcss-preset-env to v9

### Designs

_No response_

### Describe alternatives you've considered

_No response_

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### PR DESCRIPTION
**Pull request by Antonio-Laguna**
Related Issue/RFC: #326

Closes #326 #321  
### Description of the Change

Upgrade PostCSS to V9. I've also updated some dependencies to avoid security issues. I was getting `5 high severity vulnerabilities` and now it's down to `found 0 vulnerabilities`. I think this also fixes the prettier issue for good.

### Checklist:

- [X] I have read the [**CONTRIBUTING**](https://github.com/10up/10up-toolkit/blob/develop/CONTRIBUTING.md) document.
- [X] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my change.
- [ ] All new and existing tests passed.
- [X] I have added a changeset to my PR. See [**CONTRIBUTING**](https://github.com/10up/10up-toolkit/blob/develop/CONTRIBUTING.md) document for instructions

### CODE REVIEW

**Comment by Antonio-Laguna** [comment]
@nicholasio I'm unsure what's going on here, failing on a test within /projects/library. I think this is due to NPM doing funky things with installation since nothing has changed there

**Comment by nicholasio** [comment]
@Antonio-Laguna fixed, anything else that needs to be done here?

File: packages/toolkit/UPGRADING.md [hunk 1]
```diff
@@ -2,6 +2,20 @@
 
 ## Upgrading from 5 to 6
 
+There are a few major changes in this release. While it probably won't break your build, it's worth noting everything that's changed.
+
+* Node version required of 18 (previous LTS)
+* Swapping `postcss-editor-styles` with `postcss-editor-styles-wrapper` which is compatible with PostCSS 8. This should be a seamless change as this is a fork of the original package that was updated to be compatible with PostCSS 8 and fixes wrong behaviour in some circumstances.
```
Comment (nicholasio) [inline packages/toolkit/UPGRADING.md]: This is a breaking change for those extending the postcss.config.js file so they will need to update it.

**Review by nicholasio — APPROVED** [review]

**Comment by nicholasio** [comment]
@Antonio-Laguna updated a few things, was there anything else here before we merge this?

File: packages/toolkit/UPGRADING.md (hunk 1 above)
Comment (Antonio-Laguna) [reply packages/toolkit/UPGRADING.md]: Good catch

**Comment by Antonio-Laguna** [comment]
@nicholasio no, all good here! Thanks for giving it a final push. I was feeling stuck with this!

**Pull request merged**